main.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const readline = require('readline');
+var filaName = 'users.csv'
+const fileStream = fs.createReadStream(filaName);
+
+var interfaceReader = readline.createInterface({
+    input: fileStream
+});
+
+
+async function readFromFile(){
+    users = [] ;
+    for await (const line of interfaceReader) {
+       
+       const temp = line.split(",")
+       users.push(temp)
+       //console.log(temp)
```

Comment:
Let's keep our code clean, please remove this log

Reply:
Done